torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -179,6 +179,7 @@ class MixedPrecision:
         are checkpointed in their full precision (and then restored back to
         to their reduced precision) as expected. Note that this checkpoint
         support is currently limited to ``StateDictType.FULL_STATE_DICT``.
+        ``keep_casted_gradients``: Keep casted gradients.
```

Comment:
small nit (grammar): It appears the past tense conjugation of "cast" is "cast", not "casted".
- https://writingexplained.org/cast-vs-casted (search for "What does casted mean?")
- https://pasttenses.com/cast-past-tense

I am indifferent to `keep_cast_gradients` (emphasizing we are keeping the gradients, which happen to have been cast) versus `keep_gradients_cast` (emphasizing that we are keeping them cast).

Reply:
Okay I have been saying `keep_casted_gradients`, `keep_cast_gradients`, `keep_gradients_cast`, and `keep_gradients_casted` in my head, and none of them particularly sound correct. Feel free to go with whichever you prefer :)

Reply:
Interesting, yeah keep_cast_gradients sounds kinda strange to me as well.